templates/article_page.hbs

```diff
@@ -250,7 +250,9 @@
 
                             {{#if (compare (calc author.badges.length "-" 4) ">" 0)}}
                               <div class="community-badge community-badge-achievements">
-                                {{#link "user_profile" id=author.id filter_by="badges" class="community-badge-achievements-rest"}}{{t 'plus_more' count=(calc author.badges.length "-" 4)}}{{/link}}
+                                <a href="{{page_path "user_profile" id=author.id filter_by="badges"}}" class="community-badge-achievements-rest"  aria-label="{{t 'more_awards_to' name=author.name}}">
+                                  {{t 'plus_more' count=(calc author.badges.length "-" 4)}}
```

Comment:
Shouldn't the `aria-label` contain the count as well? 
I'm asking since `aria-label` replaces the content of the link in the accessibility tree. So assistive technology would read out something along the lines of "more awards to Jakub Kopys" instead of "3 more awards to Jakub Kopys". So since the number that is visible on the page would not be read, users with low visibility might be confused by the difference between what is read and displayed visually.

Reply:
I've followed the recommendation from the task (A11Y team?) - looks like they considered it good enough? But tbh I think that's a great suggestion, I will look into that